frontend/mr-admin-flate/src/components/tiltaksgjennomforinger/TiltaksgjennomforingsRad.tsx

```diff
@@ -0,0 +1,50 @@
+import { BodyShort } from "@navikt/ds-react";
+import { Tiltaksgjennomforing } from "mulighetsrommet-api-client";
+import { formaterDato } from "../../utils/Utils";
+import styles from "../listeelementer/Listeelementer.module.scss";
+import { ListeRad } from "../listeelementer/ListeRad";
+
+interface Props {
+  tiltaksgjennomforing: Tiltaksgjennomforing;
+}
+
+export function TiltaksgjennomforingsRad({ tiltaksgjennomforing }: Props) {
+  return (
+    <ListeRad
+      linkTo={`/tiltaksgjennomforinger/${tiltaksgjennomforing.id}`}
+      classname={styles.listerad_tiltaksgjennomforing}
+      testId="tiltaksgjennomforingrad"
+    >
+      <BodyShort
+        aria-label={`Navn på tiltaksgjennomforing: ${tiltaksgjennomforing.navn}`}
+      >
+        {tiltaksgjennomforing.navn}
+      </BodyShort>
+      <BodyShort
+        aria-label={`Tiltaksnummer: ${tiltaksgjennomforing.tiltaksnummer}`}
+      >
+        {tiltaksgjennomforing.tiltaksnummer}
+      </BodyShort>
+
+      <BodyShort>{tiltaksgjennomforing.virksomhetsnavn}</BodyShort>
+      <BodyShort>{tiltaksgjennomforing.tiltakstype.navn}</BodyShort>
```

Comment:
ta med aria-label for disse kolonnen også, så brukere av skjermlesere forstår konteksen